### PR TITLE
PR for quiz page

### PR DESCRIPTION
**Pull request by kiranpratyush**
1. Removed multiple questions from the Quiz Page
2. Moved Page to middle

https://user-images.githubusercontent.com/77445957/162562246-91234756-2b2d-4858-b74c-0cf381bfa0b4.mp4

### CODE REVIEW

**Review by manojasarsa — APPROVED** [review]
Good to merge

File: CSS/QUIZPAGE/style.css [hunk 1]
```diff
@@ -1,11 +1,9 @@
 main {
+  height:100%;
   display: flex;
   flex-direction: column;
   justify-content: center;
   align-items: center;
-  /* padding-left: 10px;
-  padding-right: 10px;
```
Comment (Sen-442b) [inline CSS/QUIZPAGE/style.css:7]: You can remove the comments

File: HTML/QUIZ PAGE/index.html [hunk 2]
```diff
@@ -33,43 +33,6 @@
           <div class="option-name">Option 4</div>
         </button>
       </div>
-      <div class="question-list">
-        <div class="question-name">What is the full form of CSS ?</div>
-        <button class="option option1">
```
Comment (Sen-442b) [inline HTML/QUIZ PAGE/index.html:38]: Suggestion:- you can add your options inside the `ul` and `li` elements  for better accessibility 
(don't do it if it affects the css stylings  in any way )

**Review by Sen-442b — APPROVED** [review]
- Precise CSS
- Consistent and descriptive classNames 
-  Good job with the  animations and UI

**Review by Shrinkhlarajpoot — APPROVED** [review]

File: CSS/QUIZPAGE/style.css (hunk 1 above)
Comment (kiranpratyush) [reply CSS/QUIZPAGE/style.css:7]: Thank you 
The comment is already removed. Red Lines are the removals

**Pull request merged**